src/mixins/PerItem.js

```diff
@@ -22,6 +22,26 @@ const PerItemMixin = types
     },
   }))
   .actions(self => ({
+    _validatePerItem() {
+      const objectTag = self.toNameTag;
+
+      return self.annotation.regions
+        .every((reg) => {
+          const result = reg.results.find(s => s.from_name === self);
+
+          if (!result || !result.hasValue) {
```

Comment:
this should give us the same outcome
```suggestion
          if (!result?.hasValue) {
```